composer.json

```diff
@@ -15,9 +15,12 @@
     ],
     "minimum-stability": "RC",
     "require": {
-        "php": ">=5.6.0 <8.0",
+        "php": "^7.1",
```

Comment:
Should we default this to something higher?

Reply:
All laminas dependencies support PHP 5.6, so it would be the best to support PHP 5.6 here too.